controllers/admin.js

```diff
@@ -0,0 +1,119 @@
+import models from '../models';
+import hasher from '../helpers/hasher';
```

Comment:
> This file is not auth as it only generates a hash and changing it will have to happen after your branch is merged because I presume you the way you used this function is different

Though but its part of the authentication cycle, check my branch in the helper folder and check the `auth.js`